cmd/server/cmd/crd.go

```diff
@@ -104,19 +104,33 @@ type crdVar struct {
 	Version    string
 }
 
-func newCrdVar(shrtName, implName, pluralName, label string) *crdVar {
+// pluralize gives a plural the way k8s like it.
+// https://github.com/kubernetes/gengo/blob/master/namer/plural_namer.go
+func pluralize(singular string) string {
+	switch string(singular[len(singular)-1]) {
```

Comment:
Consider adding a test for these cases in crd_test.

Reply:
Added test data for pluralization.